server/lib/tuist_web/router.ex

```diff
@@ -266,6 +266,8 @@ defmodule TuistWeb.Router do
         assigns: %{caching: not Tuist.Environment.test?(), cache_ttl: to_timeout(minute: 1)} do
     pipe_through [:open_api, :authenticated_api, :on_premise_api]
 
+    get "/accessible-projects", AccessibleProjectsController, :index
```

Comment:
wonder if we should align with GitHub's API on this? https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-the-authenticated-user

They have `/user/repositories`. That way, we can scope all endpoints for getting accessible entities under a single scope.